Modules/Core/SpatialObjects/test/itkImageMaskSpatialObjectTest4.cxx

```diff
@@ -162,18 +163,22 @@ int Test3dImageMask()
   std::cout << "   " << point << " isInside?  : "
     << maskSpatialObject->IsInsideInWorldSpace(point) << std::endl;
 
-  if(    itk::Math::NotAlmostEquals(maskBounds[0], 7.0)
-      || itk::Math::NotAlmostEquals(maskBounds[1], 8.0)
-      || itk::Math::NotAlmostEquals(maskBounds[2], 7.0)
-      || itk::Math::NotAlmostEquals(maskBounds[3], 8.0)
-      || itk::Math::NotAlmostEquals(maskBounds[4], 7.0)
-      || itk::Math::NotAlmostEquals(maskBounds[5], 8.0) )
+  for (unsigned i = 0; i < MaskBoundsArrayType::Length; ++i)
+  {
+    constexpr std::array<double, MaskBoundsArrayType::Length> expectedBounds
```

Comment:
Shouldn't this be declared outside of the for loop?

Reply:
The array is evaluated entirely at compile-time (`constexpr`), so it should not harm the performance in any way. I just placed declared the array inside, to reduce the scope (to avoid _possible_ future name clashes). However, if you like it better outside the `for` loop, no problem. I can move it out. I may need to add more curly braces to the initializer list any way (on the next few lines), to avoid some annoying compiler warning.

Reply:
@dzenanz  OK, I'll declare the array outside the for loop! With the next amend!